BoxContentSDK/BoxContentSDK/BOXURLSessionManager.m

```diff
@@ -135,7 +143,9 @@ - (NSURLSession *)createDefaultSession
         //arbitrary maxConcurrentOperationCount given that the number should not go above
         //the max number of concurrent Box api operations
         NSURLSessionConfiguration *sessionConfig = [NSURLSessionConfiguration defaultSessionConfiguration];
-
+        if (self.protocolClasses != nil) {
```

Comment:
they could but that won't affect our logic. sessionConfig.protocolClasses could be nil, and will be equal to self.protocolClasses in that case.